pixeltable/catalog/table.py

```diff
@@ -142,6 +142,15 @@ def column_types(self) -> Dict[str, ts.ColumnType]:
         """Return the names of the columns in this table."""
         return {c.name: c.col_type for c in self.tbl_version_path.columns()}
 
+    def get_attrs(self) -> schema.TableAttributes:
```

Comment:
schema.py is also not user-facing.

maybe just a get_attr(attr_name)? or no getter at all, and just comment() and num_retained_versions() instead?

Reply:
Hmm... I can see an argument for a user-facing TableAttributes struct. Saying "it's part of schema.py, so we shouldn't have a user-facing TableAttributes struct" seems like putting the cart before the horse.

get_attr(attr_name) adds user friction because they have to remember all the attribute names, and separate methods for each attribute seems less clean and doesn't scale as well.

Two other options:
1. simply move TableAttributes out of schema.py (and into table.py, say, or somewhere else) (my preference)
2. have a separate struct that's equivalent to TableAttributes but is user-facing, and table_version.py does the conversion